### PR TITLE
fix: fix minor bugs in dashboard pages (#1629)

### PR DESCRIPTION
**Pull request by yuda110**
* chore: edit sidebar title style

* fix: fix style bug in view modal

* fix: fix dashboard search bug after adding labels

---------

### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue [hunk 1]
```diff
@@ -90,6 +90,8 @@ import {
 } from 'vue';
 import { useRouter } from 'vue-router';
 
+import { store } from '@/store';
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue:93]: issue: In Vue instance, please use 'useStore'

**Review by WANZARGEN — CHANGES_REQUESTED** [review]
Please check my comment!

File: apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue:93]: oh I'll change it, thanks!

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Pull request merged**